<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AlignDialog</class>
 <widget class="QWidget" name="AlignDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Alignment</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="OKWidget" native="true">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>124</y>
     <width>288</width>
     <height>70</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(218, 218, 218);</string>
   </property>
   <widget class="QPushButton" name="btn_determine">
    <property name="geometry">
     <rect>
      <x>84</x>
      <y>15</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border-image: url(:/Dialog/Image/Dialog/determine.png);
	border:none;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="DraggableWidget" name="header" native="true">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>5</y>
     <width>290</width>
     <height>40</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent</string>
   </property>
   <widget class="QPushButton" name="close">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>5</y>
      <width>26</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/Dialog/Image/Dialog/close.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>5</y>
     <width>290</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(69, 69, 69);
</string>
   </property>
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>81</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	font: 14px &quot;Arial&quot;;
	color: rgb(255, 255, 255);
}</string>
    </property>
    <property name="text">
     <string>Alignment</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="background" native="true">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>5</y>
     <width>290</width>
     <height>189</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#background{
	background-color: rgb(227, 227, 227);
}</string>
   </property>
   <widget class="QComboBox" name="alignTo">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>67</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox{
	border:none;
	font: 75 14px &quot;Arial&quot;;
    
	color: rgb(255, 255, 255);
	border-image: url(:/Tabwidget/Image/Tabwidget/drop_down_list_background.png);
	border-left-width: 6px;
	border-right-width: 6px;

	padding-left: 4px;
}
QComboBox:drop-down{
	border-style:none;
	height:30px;
	width:18px;
	border-image: url(:/Tabwidget/Image/Tabwidget/DownArrow.png);
}</string>
    </property>
    <property name="currentText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelAlign">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>67</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>-1</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14px &quot;Arial&quot;;
color: rgb(51, 51, 51);</string>
    </property>
    <property name="text">
     <string>Align to:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="isStick">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>67</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14px &quot;Arial&quot;;
color: rgb(51, 51, 51);</string>
    </property>
    <property name="text">
     <string>Stick</string>
    </property>
   </widget>
   <widget class="QLabel" name="Information">
    <property name="geometry">
     <rect>
      <x>34</x>
      <y>100</y>
      <width>220</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14px &quot;Arial&quot;;
color: rgb(51, 51, 51);</string>
    </property>
    <property name="text">
     <string>Please select up to 2 mesh faces.</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <zorder>background</zorder>
  <zorder>widget</zorder>
  <zorder>OKWidget</zorder>
  <zorder>header</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DraggableWidget</class>
   <extends>QWidget</extends>
   <header>draggablewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
